<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 242, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="l_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Segoe UI Black&quot;;
color: rgb(2, 16, 29);</string>
    </property>
    <property name="text">
     <string>Не забывайте, что результаты данного теста недостаточно точны, 
и оценить ваши умственные способности могут
только квалифицированные специалисты.  
Тест создан в развлекательных целях.</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_text_result">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>22</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Segoe UI Black&quot;;
color: rgb(255, 43, 89);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_text_result_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 11pt &quot;Segoe UI Black&quot;;
color: rgb(2, 16, 29);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
